src/WebServer_DevicesPage.ino

```diff
@@ -565,6 +614,14 @@ void handle_devicess_ShowAllTasksTable(byte page)
             String html;
             html += F("<BR>GPIO-");
             html += Settings.TaskDevicePin3[x];
+            if (spi_gpios[0] == Settings.TaskDevicePin3[x]
+              || spi_gpios[1] == Settings.TaskDevicePin3[x]
+              || spi_gpios[2] == Settings.TaskDevicePin3[x]
+              || Settings.Pin_i2c_sda == Settings.TaskDevicePin3[x]
+              || Settings.Pin_i2c_scl == Settings.TaskDevicePin3[x]) {
+              html += " ";
```

Comment:
Please use `+= ' ';` instead of `" "` as the last one creates a string in memory compared to a single char.